go/teams/chain.go

```diff
@@ -207,6 +207,13 @@ func (t TeamSigChainState) GetUserLastJoinTime(user keybase1.UserVersion) (time
 	return t.inner.GetUserLastJoinTime(user)
 }
 
+// NewStyle invites are completed in the `used_invites` field in the change
+// membership link, can optionally specify an expiration time, and a maximum
+// number of uses (potentially infinite).
+func IsNewStyleInvite(invite keybase1.TeamInvite) (bool, error) {
```

Comment:
Why does this return an error?

Reply:
I left it as a function so we could change it if we wanted. To e.g. use a opaque "schema number" or whatever.